src/Shift-ClassInstaller-Tests/ShClassInstallerTest.class.st

```diff
@@ -288,6 +287,30 @@ ShClassInstallerTest >> testDuplicatedInstanceVariableInSuperclassMetaclass [
 	self deny: ((superClass class slots collect: [:e | e name]) includes: 'aSlot')
 ]
 
+{ #category : #tests }
+ShClassInstallerTest >> testInstallInSpecificEnvironment [
+
+	| newEnvironment |
+	newEnvironment := SystemDictionary new.
+	newClass := ShiftClassInstaller new
+		            installingEnvironment: (ShSmalltalkGlobalsEnvironment new
```

Comment:
If this is a "wrapper" as class comment in #ShGlobalsEnvironment says, why not have a 

```
ShSmalltalkGlobalsEnvironment wrapping: newEnvironment
``` 

**instance creation method** #wrapping: in #ShGlobalsEnvironment class like

```Smalltalk
wrapping: anEnvironment

	^ (self new)
			environment: anEnvironment;
			yourself
```